main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.68" # object_ownership = "BucketOwnerEnforced" supported 1st here
```

Comment:
Is this required to use this feature? 
otherwise, this is a braking change on module compatibility (there may be a few repos we don't own that use this)

Reply:
Yes this is a breaking change, because there has been changes to terraform syntax, so in order to use object lock we need the higher version of provider. These modules are versioned so existing pins should be fine.